samples/lulesh/lulesh_flux_spec_v2.0.yaml

```diff
@@ -10,10 +10,10 @@ env:
         outfile: $(SIZE.label).$(ITERATIONS.label).log
 
     dependencies:
-      git:
-        - name: LULESH
-          path: $(OUTPUT_PATH)
-          url: https://github.com/LLNL/LULESH.git
+      LULESH:
```

Comment:
This format is just my suggested one from my previous comment. I just wanted to put it here to have it in the spec to see how it looks. @jsemler and @kcathey -- thoughts?

Reply:
The indentation looks inconsistent with the rest of the `env` section. Otherwise, it looks good.